contract/test/Bank.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { ethers, upgrades } from 'hardhat'
+import { expect } from 'chai'
+import { Bank, Tips } from '../typechain-types'
+import { SignerWithAddress } from '@nomicfoundation/hardhat-ethers/signers'
+import { ContractTransactionResponse } from 'ethers'
+
+describe('Bank', () => {
+  let bank: Bank
+  let tips: Tips
+
+  async function deployContract(owner: SignerWithAddress) {
+    const TipsImpl = await ethers.getContractFactory('Tips')
+    tips = (await upgrades.deployProxy(TipsImpl, [], {
+      initializer: 'initialize'
+    })) as unknown as Tips
+
+    const BankImpl = await ethers.getContractFactory('Bank')
+    bank = (await upgrades.deployProxy(BankImpl, [await tips.getAddress()], {
+      initializer: 'initialize',
+      initialOwner: await owner.getAddress()
+    })) as unknown as Bank
+  }
+
+  // As a company owner
+  // I want to deploy/create a team's bank contract
+  // so that I can operate transaction with it
+  describe('Deploy a bank contract', () => {
+    context("Given an owner and a team's bank contract", () => {
+      let owner: SignerWithAddress
+
+      before(async () => {
+        owner = (await ethers.getSigners())[0]
+      })
+
+      context('When I deploy a bank contract', async () => {
+        before(async () => {
+          await deployContract(owner)
+          expect(await bank.getAddress()).to.be.string
+          expect(await tips.getAddress()).to.be.string
+          expect(await bank.tipsAddress()).to.eq(await tips.getAddress())
+        })
+
+        it('Then I can operate transaction with it', async () => {
+          const amount = ethers.parseEther('1')
+          const bankAddress = await bank.getAddress()
+          await owner.sendTransaction({ to: bankAddress, value: amount })
+          expect(await ethers.provider.getBalance(bankAddress)).to.eq(amount)
+        })
+
+        it('Then I owned the contract', async () => {
+          expect(await bank.owner()).to.eq(await owner.getAddress())
+        })
+      })
+    })
+  })
+
+  // As a company owner of a digital service provider.
+  // I want to direct money from my online operations to my team’s crypto “bank” account on the CNC portal.
+  // When I deposit into my team's bank contract
+  // Then I can see funds in the team's bank contract
+
```

Comment:
It's from there i remark you are using the user Stories i create for the CNC portal to make the test. 
Let me tell you that there is a difference between the userStories in the context of the CNC and in the context of the Bank Smart contract